02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  background-color: #66ff99;
+}
+
+#nameBar {
+  width: 100%;
+  background-color: #66ff99;
+  top: 0;
+  position: fixed;
+}
+
+#contactBar {
+  width: 100%;
+  background-color: #66ff99;
+  bottom: 0;
```

Comment:
Thanks, Eddie! It really is, it took me a little while to get it working
correctly and I still don't fully understand it but it sure is powerful.
See ya tonight!

On Tue, Aug 2, 2016 at 12:07 PM, Eddie Garcia <notifications@github.com>
wrote:

> In 02LessonTwo/homework/css/style.css
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/89#discussion_r73196067>
> :
>
> > @@ -0,0 +1,73 @@
> > +body {
> > +  background-color: #66ff99;
> > +}
> > +
> > +#nameBar {
> > +  width: 100%;
> > +  background-color: #66ff99;
> > +  top: 0;
> > +  position: fixed;
> > +}
> > +
> > +#contactBar {
> > +  width: 100%;
> > +  background-color: #66ff99;
> > +  bottom: 0;
>
> this code looks amazing! great job. Position: fixed; is a great tool.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/89/files/8f6273357db195e8c6180fabbe019682bf8dfa79..9d62ef412639785a46cd678ba9bcdae5f2e64105#r73196067>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIFNMX0sj678onv2h485Tb4N-n21iU2Qks5qb3lMgaJpZM4JVvQf>
> .
>